src/itemlistformaction.cpp

```diff
@@ -1120,6 +1120,12 @@ void ItemListFormAction::init()
 	if (cfg->get_configvalue_as_bool("goto-first-unread")) {
 		jump_to_next_unread_item(true);
 	}
+
+	// This is a hack to make `prepare()` do all the work it's required to do
+	// on the first run. Yes, we have a call to `invalidate_list() ` just a few
+	// lines prior, but `draw_items()` above resets the mode back to `NONE`,
+	// leading to https://github.com/newsboat/newsboat/issues/1385
+	invalidate_list();
```

Comment:
Feel free to ignore my remark:
I'm not completely sure why the items are being drawn during the init call so it might be safer to keep the current changes (especially if you want to create a new release soon).

Reply:
That'd be ideal, but we need to draw the items in `init()` for `jump_to_next_unread_item()` to work: https://github.com/newsboat/newsboat/commit/801459860dd33d9c267ed3e690f0e6e960e36c2b So I'll keep the comment and the kludge for now.